python/ray/_private/function_manager.py

```diff
@@ -151,11 +155,10 @@ def export(self, remote_function):
                                  "remote function", self._worker)
         key = (b"RemoteFunction:" + self._worker.current_job_id.binary() + b":"
                + remote_function._function_descriptor.function_id.binary())
-        if self._worker.redis_client.exists(key) == 1:
-            return
-        self._worker.redis_client.hset(
-            key,
-            mapping={
+        with disable_client_hook():
```

Comment:
I think it's because internal kv is supposed to support client and server. 

Maybe, in this case, calling gcs client directly is better, but then, it's not unified to use internal kv.